markdown/renderer.go

```diff
@@ -70,6 +70,11 @@ func (mr *Renderer) Render(w io.Writer, source []byte, node ast.Node) error {
 }
 
 func (r *render) renderNode(node ast.Node, entering bool) (ast.WalkStatus, error) {
+	// Autolink should be treated as normal string.
+	if a, ok := node.(*ast.AutoLink); ok {
+		node = ast.NewString(a.Label(r.source))
+	}
```

Comment:
Prefer not to double-process the node like this.  Fits better in the type switch:
```go
	case *ast.AutoLink:
		if entering {
			_, _ = r.w.Write(tnode.Label(r.source))
		}
```

Reply:
Make, sense. will fix